### PR TITLE
Add to readme

### PR DESCRIPTION
**Pull request by ATorrise**
Adding an additional link to our read me front and center to benefit both newer devs and clients.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ This repository also contains the Zowe Node Client SDK. The SDK lets you leverag
 <br/>
 
 ## **Documentation**
-For detailed information about how to install, configure, and use Zowe CLI, see [Zowe CLI Documentation](https://docs.zowe.org/stable/getting-started/cli-getting-started/). The documentation includes examples and tutorials for how to contribute to Zowe CLI and develop CLI plug-ins.
+For detailed information about how to install, configure, and use Zowe CLI, see [Zowe CLI Quickstart Documentation](https://docs.zowe.org/stable/getting-started/cli-getting-started/) or the more thorough [Zowe CLI Documentation](https://docs.zowe.org/stable/user-guide/cli-using-usingcli/). The documentation includes examples and tutorials for how to contribute to Zowe CLI and develop CLI plug-ins.
```
Comment (t1m0thyj) [inline README.md:24]: Should we make the same change to `packages/cli/README.md`? People only see the top-level one when they browse our repo on GitHub: https://github.com/zowe/zowe-cli. The readme inside `packages/cli` is the one shown on npm: https://www.npmjs.com/package/@zowe/cli

Comment (ATorrise) [reply README.md:24]: Yes whoops

Comment (anaxceron) [inline README.md:24]: I'd try a little rewording: 

For information about how to install, configure, and use Zowe CLI, see [Zowe CLI Quickstart Documentation](https://docs.zowe.org/stable/getting-started/cli-getting-started/). For more detailed instructions, see [Zowe CLI Documentation](https://docs.zowe.org/stable/user-guide/cli-using-usingcli/). 

Not sure if this sentence -- "The documentation includes examples and tutorials for how to contribute to Zowe CLI and develop CLI plug-ins." -- refers to the Quick Start or the more detailed instructions or both.

If it refers to the more detailed stuff, then I'd incorporate it like this: 

For more detailed instructions, see [Zowe CLI Documentation](https://docs.zowe.org/stable/user-guide/cli-using-usingcli/), which includes examples and tutorials for how to contribute to Zowe CLI and develop CLI plug-ins.

Also: We use "quick start" not "quickstart"

**Review by anaxceron — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ATorrise) [reply README.md:24]: Thank you!

**Pull request merged**